src/ansi-colors-browser.ts

```diff
@@ -16,21 +16,24 @@ export const ansiColorsLog = (...input: unknown[]) => {
       continue;
     }
 
-    str += segment.replace(ansiRegex(), (escapeCode) => {
-      // \u001b is unicode for <ESC>
-      // eslint-disable-next-line no-control-regex
-      const parsedEscapeCode = /\u001B\[(\d*)m/.exec(escapeCode);
-      if (!parsedEscapeCode) return ''; // Unrecognized escape code, remove it
-      const escapeCodeNum = Number(parsedEscapeCode[1]); // Capture group
-      const cssObj = ansiCodes[escapeCodeNum];
-      if (!cssObj) return '';
-      Object.assign(styleStack, cssObj);
-      const cssStr = Object.entries(styleStack)
-        .map(([key, val]) => (val === 'inherit' ? '' : `${key}: ${val};`))
-        .join('');
-      stylesAndSubstitutions.push(cssStr);
-      return '%c';
-    });
+    str += segment
+      // Convert back from "next-line" character into regular newline for display in browser
+      .replace(/\u0085/g, '\n')
```

Comment:
> @gerardo-rodriguez It looks like the unconventional newline character doesn't get displayed in CI on GitHub. Starting to think maybe we should remove it

@calebeby Is this the line for the unconventional newline character? I can't remember it's been so long now. 😅

Reply:
Yeah this is one place where it is, I think there is another place too but I can't find it rn because I'm on my phone

Reply:
Thanks! I can look. 🔍 👀